pom.xml

```diff
@@ -16,6 +16,7 @@
 
 	<properties>
 		<java.version>1.8</java.version>
+        <oauth2.version>2.2.1.RELEASE</oauth2.version>
```

Comment:
This comment should be in line 7 :P :
What this **spring-boot-starter-parent** dependency means?

Reply:
The spring-boot-starter-parent dependency is the parent POM which provides dependency and plugin management for Spring Boot-based applications